manifests/init.pp

```diff
@@ -52,6 +54,7 @@
 #
 class foreman_proxy_content (
   String[1] $parent_fqdn = $foreman_proxy_content::params::parent_fqdn,
+  String[1] $foreman_fqdn = $foreman_proxy_content::params::foreman_fqdn,
```

Comment:
It will definitely break the installer, as the scenarios include foreman_proxy with parameters.